sdk/core/azure-core-test-v2/src/main/java/com/azure/v2/core/test/TestBase.java

```diff
@@ -0,0 +1,371 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.v2.core.test;
```

Comment:
Can we make this a clientcore package or is this very Azure specific testing framework that cannot be generalized?

Reply:
Test proxy is a very Azure SDK team concept, and that's most of the value of TestBase. I think it has to stay in a branded package.

Reply:
This wasn't even supposed to be part of this PR. I may have accidentally branched from another branch rather than main for this PR.